scripts/scil_tractogram_filter_by_roi.py

```diff
@@ -75,28 +101,28 @@ def _build_arg_parser():
     p.add_argument('out_tractogram',
                    help='Path of the output tractogram file.')
 
-    p.add_argument('--drawn_roi', nargs='+', action='append',
+    p.add_argument('--drawn_roi', nargs='+', action='append', default=[],
```

Comment:
isn't `'append'` the default `action` for `nargs='+'`?

Reply:
No, appends make it so that you can call the same option many times:
`--draw_roi option1 option2  --draw_roi option3 option4`

And the result will be `args.drawn_roi=[[option1, option2], [option3, option4]]`

Reply:
ah ok I get it